sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 1.4.0-beta.4 (Unreleased)
 
+### Breaking Changes
+- Update the default value of `ExcludeSharedTokenCacheCredential` on `DefaultAzureCredentialsOptions` to true, to exclude the `SharedTokenCacheCredential` from the `DefaultAzureCredential` by default.
```

Comment:
Did you intend to reference the BREAKING_CHANGES.md here?

Reply:
I do want to reference it but I believe it should be an absolute link since this the changelog gets put into the nupkg. I was planning on doing this as a separate change unless you have a better way to accomplish this.

Reply:
I think the link checker does the right thing with the absolute links. It may just work.